packages/anchor/src/components/AddLinkForm.tsx

```diff
@@ -45,13 +45,7 @@ const AddLinkForm = (props: AddLinkFormParams): ReactElement => {
 
   const onChange = (event: ChangeEvent<HTMLInputElement>): void => {
     const newValue = event.target.value;
-
-    if (isInvalid && isUrl(URLUtils.normalizeUrl(newValue))) {
```

Comment:
I don't understand why this condition was like this: in the case if the current state `isInvalid` then we for sure want to recheck if the newly url is now valid, in the case when the url was `valid` we also need to change (on each change) if the new value is valid.... or did I miss something?

Reply:
looks like it was already wrong before though

```
    if (this.state.isInvalid && URLUtils.isUrl(URLUtils.normalizeUrl(value))) {	
      nextState.isInvalid = false;	
    }	
    this.setState(nextState);
```

let's assume the value was `https://google.ch` (valid) so `this.state.isInvalid` was `false` now the next change is `https:/google.ch` (removed a slash). It will now still be seen as valid, since the check is never executed.

But anyway I think it's correct now :)